src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -188,18 +165,23 @@ private static String generateHeader(String pluginName){
      *
      * @return String  total documentation for the page
      */
-    public static String generatePluginHelp(String pluginName, Map<String, List<StepDescriptor>> byPlugin, boolean genHeader){
+    public static String generatePluginHelp(String pluginName, String displayName, Map<String, List<StepDescriptor>> byPlugin, boolean genHeader){
         Main.isUnitTest = true;
 
         //TODO: if condition
         StringBuilder whole9yards = new StringBuilder();
         if(genHeader){
-            whole9yards.append(generateHeader(pluginName));
+            whole9yards.append(generateHeader(displayName));
         }
         
-        whole9yards.append("== ").append(pluginName).append("\n\n");
+        whole9yards.append("== ").append(displayName).append("\n\n");
```

Comment:
We might want to keep the anchors in here using the `pluginName`. 
Add a `[[<pluginName]]` above these?